test/plugin/run.sh

```diff
@@ -93,9 +104,15 @@ waitForAvailable() {
         exitAndClean 1
     fi
 }
-parse_commandline "$@"
 
 start_stamp=`date +%s` ### start
+parse_commandline "$@"
+
+if [[ ! -d ${agent_home} ]]; then
+    echo "[WARN] SkyWalking Agent not exists"
+    ${mvnw} -f ${home}/../../pom.xml -Pagent -DskipTests clean package 
```

Comment:
Are you trying to build agent jar if the prev step didn't execute?

Reply:
Yes.
It is trigger only when it run local.